src/foam/lib/json/Outputter.java

```diff
@@ -38,6 +38,7 @@ protected SimpleDateFormat initialValue() {
   protected boolean       outputShortNames_    = false;
   protected boolean       outputDefaultValues_ = false;
   protected boolean       outputClassNames_    = true;
+  protected boolean       outputReadableDates_ = true;
```

Comment:
Does everything still work if this is false?

Reply:
Yes.

This flag is so that cronjobs appear in the journals as readable strings in case people are editing by hand or reading the journal by hand.  I'd be happy to remove it if we don't care about that use case and always serialize as numbers.